apps/web/src/services/asset-inventory/components/AddCustomMetricModal.vue

```diff
@@ -18,24 +30,37 @@ const props = withDefaults(defineProps<Props>(), {
 });
 
 const emit = defineEmits<{(e: 'update:visible', value: boolean): void}>();
+const metricExplorerPageStore = useMetricExplorerPageStore();
+
+const storeState = reactive({
+    metricList: computed(() => metricExplorerPageStore.state.metricList),
```

Comment:
suggestion: Why don't you change this to `metricNameList(string[])`?

```typescript
metricNameList: computed(() => metricExplorerPageStore.state.metricList.map((metric) => metric.name)),
```

Reply:
Good idea!